strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/storage/repository/RepositoryManagementTest.java

```diff
@@ -108,7 +108,7 @@ private void artifactShouldBeCorrectlyResolvedAndUnique(Path artifact)
         throws IOException
     {
         assertThat(Files.exists(artifact)).isTrue();
-        assertThat(Files.size(artifact)).isEqualTo(1024L);
+        assertThat(Files.size(artifact)).isGreaterThan(1024L);
```

Comment:
Just for my better understanding, why is this change required?

Reply:
Because when file is generated using `TestFileUtils.generateFile` extra bytes are written to output stream hence size of artifact increases.

Reply:
Right, thanks!